index.html

```diff
@@ -2517,26 +2513,67 @@ <h3>
         <dl>
           <dt>@context</dt>
           <dd>
-The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
-where the value of the first <a>URI</a> is
-<code>https://www.w3.org/ns/did/v1</code>. All members of the
+The value of the <code>@context</code> property MUST be exactly one of the following:
+
+<ul>
+  <li>The string <code>https://www.w3.org/ns/did/v1</code>. 
+    <pre class="example">
+      {
+        "@context": "https://www.w3.org/ns/did/v1",
+        "id": "did:example:123"
+      }      
+    </pre>
+  </li>
+  <li>An array of one or more <a>URIs</a>,
+    where the value of the first <a>URI</a> is
+    <code>https://www.w3.org/ns/did/v1</code>.
+
+    <pre class="example">
+      {
+        "@context": [
+          "https://www.w3.org/ns/did/v1",
+          "https://example.com/blockchain-identity/v1"
+        ],
+        "id": "did:example:123"
+      }      
+    </pre>
+  </li>
+  <li>An array where the first value is
```

Comment:
I'm probably missing something but can you elaborate as to why this explicit variation of the `@context` value has to be noted in the spec? For instance why it is not sufficient to purely state that the did document `@context` value is either a string where the value is `https://www.w3.org/ns/did/v1` or an array where the first member of the array MUST be a string with the value `https://www.w3.org/ns/did/v1` and impose no further constrains around what the other elements in the array must be.

Reply:
I suppose it could say, an array with first value equal to `https://www.w3.org/ns/did/v1`... I tend to favor explicit over implicit definitions, hence explicit support for the cases I have seen in the wild.

Reply:
@tplooker I tried rewording this, and it always ended up dangerously vague. Feel free to suggest changes you think will provide better clarity. 

I would prefer to resolve this conversation and keep my suggested changes to the requirements associated with `@context`, can I take that action?

Reply:
@OR13 just made a suggestion above, let me know what you think but if that works then I think this section can be removed?